Shopify/dawn
# Default Follow on Shop to Enabled

### PR Summary: 

In the footer, the `enable Follow on Shop` setting should default to true.

### Why are these changes introduced?

Fixes #2328

### Visual impact on existing themes


Assuming that the merchant meets all the criteria for Follow on Shop, the Follow on Shop button will appear in the footer.

### Testing steps/scenarios

* Navigate to the theme editor
* Go to the `footer` section
* Follow on Shop should be enabled

<img width="301" alt="Screenshot 2023-02-23 at 12 56 55 PM" src="https://user-images.githubusercontent.com/1103255/221028473-ed7c0f28-61a2-48c8-bb61-df97fda4c331.png">


### Demo links


- [Editor](https://jared-schaab.myshopify.com/admin/themes/130888499256/editor)

### Checklist
